tests/Behat/Page/Admin/OrderCreatePage.php

```diff
@@ -12,22 +12,13 @@
 {
     public function addProduct(string $productName): void
     {
-        $this->getDocument()->clickLink('Add');
-
-        $item = $this->getLastOrderItem();
+        $item =$this->addItemAndWaitForIt();
```

Comment:
REALLY MINOR ISSUE. However, it should be 
```
$item = $this->addItemAndWaitForIt();
```
rather than 
```
$item =$this->addItemAndWaitForIt();
```

of course it can be fixed in next PR.

Reply:
It will be fixed in this PR when I make tests passing finally xD

Reply:
I think there's ECS for coding standard, so it should be just running `bin/ecs check src --fix`.